test-files/article.html

```diff
@@ -76,9 +76,9 @@ <h3 class="code-title">Code</h3>
 			<h2 class="test-title" id="ar-004">Test ar-004: Article with <code>aria-labelledby</code></h2>
 
 			<div class="test-case">
-				<article aria-labelledby="ar-004-labelledby">
-					<h1 id="ar-004-labelledby">h1 referenced by aria-labelledby</h1>
-					<p>Aria-labelledby should be used as accessible name.</p>
+				<article aria-labelledby="ar-004-labelledby4">
+					<h1 id="ar-004-labelledby4">h1 referenced by aria-labelledby4</h1>
+					<p>Aria-labelledby should be used as accessible name. 4</p>
```

Comment:
These are just help text to say which should win, and not part of the test, so I don't think these need the numbers added?